### PR TITLE
Add string_inserts (EventData) to the resulting record.

### PR DESCRIPTION
**Pull request by igorpeshansky**
Also allow different value types for record keys and fix the type of the `keys` config option.
Fix typos in the README.

### CODE REVIEW

**Comment by repeatedly** [comment]
Could you update README together? https://github.com/fluent/fluent-plugin-windows-eventlog#read-keys

File: README.md [hunk 1]
```diff
@@ -19,46 +19,48 @@
...
+|`event_category`|
+|`source_name`   |
+|`computer_name` |
+|`user`          |
+|`description`   |
+|`string_inserts`|
 
 ## Etc.
-'read_from_head' is not supporeted currently.You can read newer records after you start first.
+'`read_from_head`' is not supported currently. You can read newer records after you start first.
```
Comment (igorpeshansky) [inline README.md]: This is confusing -- the code seems to support the option. Should we remove this statement from the README?

Comment (repeatedly) [inline README.md]: @cosmo0920 How about this?

Comment (cosmo0920) [inline README.md]: I checked this parameter behavior with current master.
This option is supported in current master.
We can remove this.

Comment (repeatedly) [inline README.md]: Okay. Let's remove!

Comment (igorpeshansky) [inline README.md]: Done (moved to the documented options section).

**Comment by igorpeshansky** [comment]
Addressed the feedback, please take another look.
@repeatedly Would you like me to squash the commits before you merge?

**Comment by igorpeshansky** [comment]
@repeatedly I saw that you tagged version 0.2.1 with this change — thanks. Any chance we could also get an updated gem on rubygems.org?

**Pull request merged**